datadog_checks_dev/datadog_checks/dev/tooling/commands/release/trello/tester_selector/trello_users.py

```diff
@@ -44,5 +44,5 @@ def get_users(self) -> List[TrelloUser]:
                         'full_name': fullname,
                         'username': member['username'],
                     }
-        self.__user_cache.set_value(trello_users)
+                    self.__user_cache.set_value(trello_users)
```

Comment:
Note to reviewer: `trello_users` contains the entire list of Trello users who have been processed, so this user cache keeps replacing the existing content with the new list, even though only 1 new entry is added after each call.